### DISCUSSION TITLE
How to limit the size of txt file?

### DISCUSSION THREAD

**Discussion by Youshikawa** [coder/code-server#4570] [Q&A]
Great project!
When I view a txt file that is too large ,  My server will be crashed.
In order to avoid this condition. How to I setting the txt reader?
Thx.

**Comment by jsjoeio** [comment]
Hmm...I don't think this is something you can edit unfortunately. I looked through [Settings](https://code.visualstudio.com/docs/getstarted/settings) but don't see anything around limiting size of text files. You might want to ask upstream in the microsoft/vscode repo.
